projects/cashmere/src/lib/pop/directives/popover-anchor.directive.ts

```diff
@@ -104,6 +105,15 @@ export class HcPopoverAnchorDirective implements OnInit, AfterContentInit, OnDes
         this._attachedPopover.maxWidth = val;
     }
 
+    /** Whether the popover should return focus to the previously focused element after closing. *Defaults to `true`.* */
+    @Input()
+    get restoreFocus() {
+        return this._attachedPopover.restoreFocus && this._attachedPopover._restoreFocusOverride;
+    }
+    set restoreFocus(val: boolean) {
+        this._attachedPopover.restoreFocus = coerceBooleanProperty(val);
```

Comment:
Figures that there's a CDK function for this.  We actually have our own flavor of the same thing that you should switch to for consistency with the other components.  You can import it with:
`import {parseBooleanAttribute} from '../util';`

and then the function is:
`parseBooleanAttribute(val)`